src/ker_lowupsampfac_horner_allw_loop_constexpr.h

```diff
@@ -0,0 +1,195 @@
+          {1.5672684443241270E-08, 3.5812571134853209E-07, 1.1292168823202786E-06, -2.5215449854178345E-06, -7.6275609266392180E-06, 9.3973092319735694E-06, 1.7891569285079721E-05, -1.8642776809377737E-05, -1.8642776809426295E-05, 1.7891569285078386E-05, 9.3973092319897562E-06, -7.6275609266437149E-06, -2.5215449854180594E-06, 1.1292168823201941E-06, 3.5812571134853119E-07, 1.5672684443241263E-08},
+          {3.6571939291734580E-08, 1.5742222553115409E-06, 1.1217451065775842E-05, 1.0668471374318627E-05, -6.0694020243069901E-05, -7.4268888177613095E-05, 1.3567546096380107E-04, 1.4875477215032178E-04, -1.4875477215047734E-04, -1.3567546096381663E-04, 7.4268888177613095E-05, 6.0694020243062122E-05, -1.0668471374319355E-05, -1.1217451065775811E-05, -1.5742222553115415E-06, -3.6571939291734593E-08},
+          {6.1501023800531295E-08, 4.8443034242391141E-06, 6.0167136036954503E-05, 2.0573318254802077E-04, 1.2811955521419976E-05, -8.3782209201438288E-04, -6.2669687707128208E-04, 1.1809008871738671E-03, 1.1809008871739529E-03, -6.2669687707129921E-04, -8.3782209201441725E-04, 1.2811955521421050E-05, 2.0573318254802050E-04, 6.0167136036954456E-05, 4.8443034242391132E-06, 6.1501023800531308E-08},
+          {7.2283166867263303E-08, 1.0391634193778160E-05, 2.0529674430143854E-04, 1.2618687081127932E-03, 2.6256301814801060E-03, -5.5040645592551222E-04, -7.8709464111364341E-03, -5.7657980103486698E-03, 5.7657980103487626E-03, 7.8709464111365313E-03, 5.5040645592551373E-04, -2.6256301814801060E-03, -1.2618687081127928E-03, -2.0529674430143856E-04, -1.0391634193778164E-05, -7.2283166867263303E-08},
+          {5.6049296769722387E-08, 1.4879146623074258E-05, 4.4787865139353365E-04, 4.2383440773521696E-03, 1.6624620601556193E-02, 2.6395394769117640E-02, 3.6740117889106082E-04, -4.8088574473126630E-02, -4.8088574473126713E-02, 3.6740117889113471E-04, 2.6395394769117678E-02, 1.6624620601556193E-02, 4.2383440773521722E-03, 4.4787865139353359E-04, 1.4879146623074262E-05, 5.6049296769722367E-08},
+          {2.5620581163903708E-08, 1.2815874111792787E-05, 5.7471335914300670E-04, 7.8386860177525539E-03, 4.6638901641906962E-02, 1.3897554029141571E-01, 2.0773808644544137E-01, 1.0813440420918320E-01, -1.0813440420918335E-01, -2.0773808644544145E-01, -1.3897554029141571E-01, -4.6638901641906975E-02, -7.8386860177525539E-03, -5.7471335914300648E-04, -1.2815874111792787E-05, -2.5620581163903721E-08},
+          {5.2012152104083984E-09, 5.0291159580938677E-06, 3.3201112337137920E-04, 6.3015433246683353E-03, 5.2427915343763412E-02, 2.3104762006593377E-01, 5.9521037322997217E-01, 9.4441119081353875E-01, 9.4441119081353875E-01, 5.9521037322997217E-01, 2.3104762006593377E-01, 5.2427915343763412E-02, 6.3015433246683353E-03, 3.3201112337137920E-04, 5.0291159580938694E-06, 5.2012152104083868E-09},
+      }};
+    } else {
+      static_assert(w >= 2, "w must be >= 2");
+      static_assert(w <= 16, "w must be <= 16");
+      return {};
+    }
```

Comment:
Please look at https://github.com/flatironinstitute/finufft/pull/499#discussion_r1687192225

I notices this breaks clang msvc.